packages/cli/test/fixtures/unit/commands/build/storybook-with-middleware/.vercel/project.json

```diff
@@ -2,8 +2,6 @@
   "orgId": ".",
   "projectId": ".",
   "settings": {
-    "framework": "storybook",
-    "buildCommand": "npm run build-storybook",
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** test failing build command

In this or a follow-up PR: ‏Should we leave a bad build command in here to test that the ignore is happening properly?

Reply:
If `buildCommand` is set here (when the user explicitly places an override on the build command in the UI), it takes highest precedence. What I wanted to test was that the default command (lowest precedence) is what gets executed here.